.github/workflows/pull-request.yml

```
@@ -69,6 +71,7 @@ jobs:
   icon-check:
     name: Icon Check
     runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
```

Does it need to be wrapped in {{}}

---

```suggestion
    if: ${{ github.event.pull_request.draft == false }}
```

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression, unless the expression contains any operators. If the expression contains any operators, the expression must be contained within ${{ }} to explicitly mark it for evaluation.


https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions